frontends/web/src/new_front/components/CreateSamples/CreateSamples/AnnotationInterfaces/Contexts/ChatWithInstructions.tsx

```diff
@@ -330,7 +330,9 @@ const ChatWithInstructions: FC<
                   <div className="flex items-end justify-end gap-4">
                     <GeneralButton
                       text="I understand"
-                      onClick={() => setReadInstructions(true)}
+                      onClick={() => {setReadInstructions(true);
+                        setIsGenerativeContext(false);}
```

Comment:
If an user hasn't click on save and then finish, and clicks on the submit button at the end of the page the interaction with the bot wouldn't be properly saved

Reply:
Please remove line 334